migrations/v2.js

```diff
@@ -0,0 +1,37 @@
+import { describe , whereContent, whereFromPlugin, whereToPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
```

Comment:
Picky I know but the same in both v2 and v6

```suggestion
import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
```

Reply:
I think I copied this from an example file, but will correct and only use methods that are used in the migration script.

Reply:
You should be able to use eslint inside migration scripts. It'll flag up read for stuff that isn't needed.

![image](https://github.com/user-attachments/assets/500335d5-d9fd-4073-9422-5ec84cc0a0f6)